amp/WEB-INF/src/org/digijava/module/aim/action/CompareActivityVersions.java

```diff
@@ -358,10 +358,8 @@ public ActionForward viewDifferences(ActionMapping mapping, ActionForm form, Htt
             HttpServletResponse response) throws Exception {
 
         CompareActivityVersionsForm vForm = (CompareActivityVersionsForm) form;
-        AmpActivityVersion av = ActivityUtil.loadActivity(vForm.getActivityOneId());
-        vForm.setActivityName(av.getName());
         vForm.setOutputCollectionGrouped(ActivityVersionUtil.compareActivities(vForm.getActivityOneId()));
-
+        vForm.setActivityName(ActivityVersionUtil.activityName);
```

Comment:
this is not 100% correct since activityName is a static variable and it can get mixed